### PR TITLE
Create Example_ReferenceTestBed_UserGuide

### PR DESCRIPTION
**Pull request by jfernandezsqs**

### CODE REVIEW

**Review by Sonia-IDSA — APPROVED** [review]

File: Testbed/Example_ReferenceTestBed_UserGuide.md [hunk 1]
```diff
@@ -0,0 +1,730 @@
+# Example Reference Testbed User Guide
```
Comment (ssteinbuss) [inline Testbed/Example_ReferenceTestBed_UserGuide.md:1]: We should reference [this](../Testbed/Example_ReferenceTestBed_UserGuide.md) from the README.md to guide the user through the Repository.

File: Testbed/Preparing_preconfigured_setup.md [hunk 2]
```diff
@@ -0,0 +1,600 @@
+# Preparing connector A and B
```
Comment (ssteinbuss) [inline Testbed/Preparing_preconfigured_setup.md:1]: We should reference [this](../Testbed/Preparing_preconfigured_setup.md) from the README.md to guide the user through the Repository.

**Review by ssteinbuss — APPROVED** [review]
This is a good extension. We have to link the new resources in the README.md to guide the user through the repository

File: Testbed/Example_ReferenceTestBed_UserGuide.md (hunk 1 above)
Comment (ssteinbuss) [reply Testbed/Example_ReferenceTestBed_UserGuide.md:1]: Created  a new issue for this #18

File: Testbed/Preparing_preconfigured_setup.md (hunk 2 above)
Comment (ssteinbuss) [reply Testbed/Preparing_preconfigured_setup.md:1]: Created  a new issue for this #18

**Review by mohuber — COMMENTED** [review]
Hi everyone,
I have finished my review and changed this exemplary user guide to serve as description for the preconfiguration and validation of the testbed setup it is used for. Additionally, I have added some useful information from this file to the general TestbedUserGuide.
@jfernandezsqs could you please finish alignment of this Preconfiguration Guide with the Postman collection?

**Pull request merged**